core/chains/evm/txmgr/config.go

```diff
@@ -0,0 +1,84 @@
+package txmgr
+
+import (
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	txmgrtypes "github.com/smartcontractkit/chainlink/v2/common/txmgr/types"
+	"github.com/smartcontractkit/chainlink/v2/core/assets"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/gas"
+	"github.com/smartcontractkit/chainlink/v2/core/services/pg"
+)
+
+// Config encompasses config used by txmgr package
+// Unless otherwise specified, these should support changing at runtime
+//
+//go:generate mockery --quiet --recursive --name Config --output ./mocks/ --case=underscore --structname Config --filename config.go
+type Config interface {
+	gas.Config
+	pg.QConfig
+	EthTxReaperInterval() time.Duration
+	EthTxReaperThreshold() time.Duration
+	EthTxResendAfterThreshold() time.Duration
```

Comment:
Since we are moving these anyway does it make sense to make it `EVMxxx` for consistent naming everywhere?

Reply:
i.e. rename tables `eth_tx` => `evm_tx` etc also

Reply:
eventually yes, but that would be quite the change to propagate through the whole codebase :)